eng/targets/Imports.targets

```diff
@@ -18,12 +18,6 @@
       which has all of our global settings 
     -->
     <UserRuntimeConfig Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND '$(OutputType)' == 'Exe'">$(RepositoryEngineeringDir)config\runtimeconfig.template.json</UserRuntimeConfig>
-
-    <!-- 
-      Bootstrap compiler should run as a 32 bit process on our 32 bit test runs. This will validate
-      the legacy JIT ability to handle the compiler. 
-    -->
-    <PlatformTarget Condition="'$(OutputType)' == 'Exe' AND '$(TargetFramework)' == 'net472' AND '$(BOOTSTRAP32)' == 'true'">x86</PlatformTarget>
```

Comment:
This was specifically introduced after a legacy jit regression issue. What are we doing to replicate the test?

Reply:
This was done to guard against a potential legacy JIT issue. Eventually we determined that it was just a more vanilla race condition. At the time though we were quite stumped and decided to do this as a test. The compiler is fairly well tested on the legacy JIT via our unit tests including minimal execution of csc.exe. Hence the added coverage here is very minimal and not worth the infrastructure to maintain.